Automattic/studio
# Run wp db import command when importing site backup

Related to https://github.com/Automattic/dotcom-forge/issues/8363

## Proposed Changes

This PR implements the `wp sqlite import` command for `.sql` files that are uploaded via the import feature. This is a third-party command from the [wp-cli-sqlite-command](https://github.com/Automattic/wp-cli-sqlite-command) package.

## Testing Instructions



- Start Studio via `STUDIO_IMPORT_EXPORT=true npm start` 
- Navigate to the `Studio/server-files` folder and check that a folder `sqlite-command` was created.
- Check that there is a `version` file in the `sqlite-command` folder and that it shows the latest version of the command package.
- Try to export the site and verify that a valid SQL export is placed in the export archive.
- Try to export only the database, verify that an SQL file with a valid export has been created at the chosen location.
- Try to import an SQL file and verify that the data was successfully imported.
- Try to import a full site backup that also has an SQL export and verify that all data is successfully imported

## Pre-merge Checklist
